applications/FluidDynamicsApplication/custom_utilities/mass_conservation_utility.cpp

```diff
@@ -673,14 +791,14 @@ Triangle2D3<Node<3>>::Pointer MassConservationCheckProcess::GenerateAuxTriangle(
 
 
 
-void MassConservationCheckProcess::GenerateAuxLine( const Geometry<Node<3> >& rGeom,
-                                                    const Vector& distance,
-                                                    Line3D2<IndexedPoint>::Pointer& p_aux_line,
-                                                    array_1d<double, 3>& aux_velocity1,
-                                                    array_1d<double, 3>& aux_velocity2 ){
+void MassConservationUtility::GenerateAuxLine( const Geometry<Node<3> >& rGeom,
+                                                    const Vector& rDistance,
+                                                    Line3D2<IndexedPoint>::Pointer& rpAuxLine,
+                                                    array_1d<double, 3>& rAuxVelocity1,
+                                                    array_1d<double, 3>& rAuxVelocity2 ){
 
     // Compute the relative coordinate of the intersection point over the edge
-    const double aux_node_rel_location = std::abs ( distance[0] /( distance[1]-distance[0] ));
+    const double aux_node_rel_location = std::abs ( rDistance[0] /( rDistance[1]-rDistance[0] ));
```

Comment:
Shouldn't we check that this division is not 0.0?

Reply:
hmm you enter here if the line is cut, so distances have different signs. I'd say its not needed.

Reply:
OK.